tests/acceptance/file-page-test.js

```diff
@@ -21,14 +21,15 @@ module('Acceptance | file page', (hooks) => {
 	});
 
 	test('visiting File Page', async (assert) => {
+		await visit('/');
```

Comment:
did you open ticket to fix that?

Reply:
I'm in the middle

Reply:
https://wikia-inc.atlassian.net/browse/XF-565